samples/HostTests/.cproject

```diff
@@ -0,0 +1,151 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
It would be better to keep in the `samples` directory only projects that the users can use as a base for their project and start extending it. The `HostTests` is more appropriate for a `Tests` folder. 

We should place the tests to be on the same level as `samples` and every test project should specify a list of applicable architectures to be tested on or on all of them if none is specified. Something like this

```
/_
   |_  Sming/
   |_  samples/
   |_  tests/
          |_  HostTests
          |_  ...
```
As a start you can move `HostTests` to `tests`.

Reply:
Should it be `Tests` or `tests` I wonder?

Reply:
`tests` please. I have corrected my previous post.